doc/api/vm.md

```diff
@@ -108,12 +108,12 @@ for (var i = 0; i < 10; ++i) {
   script.runInContext(context);
 }
 
-console.log(util.inspect(sandbox));
+console.log(sandbox);
 
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
-### script.runInNewContext([sandbox][, options])
+### script.runInNewContext(sandbox[, options])
```

Comment:
@davidgilbertson If you’re git-savvy enough, I’d suggest changing the commit in this PR that made the original modification; i.e. `git rebase -i master`, replacing `pick` with `edit` for cb555d106c1edec7db2fe2ac6b96c088f0c92c36, then running the rebase.

You can also add another commit if that’s easier for you; generally, any way that makes a person who lands this PR understand what’s going on should be fine. :)